deployment/k8s/base/configmap.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 data:
   config.json: |
     {
+        "kubeletRootDir": "/var/lib/kubelet",
```

Comment:
lets have this one as a cli flag i dont think users should be concerned about it.  (--kubelet-root-dir)

thats a deployment decision.

Reply:
normally, i also dont think users should be concerned about it, its default value is /var/lib/kubelet,
if necessary, users can set kubeletRootDir=xxx

Reply:
cli and configmap are two different configurations. If necessary, It can be changed to cli mode.

Reply:
> It can be changed to cli mode.

that is my preference.